models/public/efficientnet-b7_auto_aug-tf/efficientnet-b7_auto_aug-tf.md

```diff
@@ -0,0 +1,74 @@
+# efficientnet-b7-tf
+
+## Use Case and High-Level Description
+
+The `efficientnet-b7_auto_aug-tf` model is one of the [EfficientNet](https://arxiv.org/abs/1905.11946)
+group of models designed to perform image classification, trained with
+[AutoAugmentation preprocessing](https://arxiv.org/abs/1805.09501).
+This model was pretrained in TensorFlow\*.
+All the EfficientNet models have been pretrained on the ImageNet image database.
+For details about this family of models, check out the [repository](https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet).
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 77.618        |
+| MParams           | 66.193        |
+| Source framework  | TensorFlow\*  |
+
+## Accuracy
+
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| Top 1  | 84.68          | 84.68           |
+| Top 5  | 97.09          | 97.09           | 
+
+## Performance
+
+## Input
+
+### Original model
+
+Image, name - `image`,  shape - `[1x600x600x3]`, format is `[BxHxWxC]` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Channel order is `BGR`.
```

Comment:
Are you sure that `BGR`?

Reply:
As I see in internal config, you use pillow to read images. Pillow always read image in `RGB`